techniques/general/G224.html

```diff
@@ -233,39 +233,39 @@ <h2>Examples</h2>
 
 
 <section id="related">
-      <h2>Related Techniques</h2>
+      <h2>関連テクニック</h2>
       <ul>
       <li><a href="../css/C20">C20: Using relative measurements to set column widths so that lines can average 80 characters or less when the browser is resized</a></li>
       </ul>
     </section>
 <section id="tests">
-      <h2>Tests</h2>
-      <section class="procedure" id="procedure-for-list-example"><h3>Procedure for list example</h3>
+      <h2>検証</h2>
+      <section class="procedure" id="procedure-for-list-example"><h3>リストの事例の手順</h3>
          <ol>
-            <li>Display the web page in a user agent where the page can be zoomed, or the user agent can be resized.</li>
-            <li>Zoom in or resize the browser window so that the viewport is equivallent to 320 CSS pixels wide.</li>
-            <li>The list content fits within the width so that it is only necessary so scroll vertically to read the list.</li>
-            <li>Or, each nested list can be horizontally scrolled into view. Only vertical scrolling is necessary to read the content of list items within each specific nested list level.</li>
+            <li>ウェブページを、ページをズームしたり、ユーザエージェントのサイズを変更したりできるユーザエージェントで表示する。</li>
+            <li>ブラウザのウィンドウをズームイン又はサイズ変更し、ビューポートが 320 CSS ピクセル幅に相当するようにする。</li>
+            <li>リストの内容が幅に収まるように、リストを読むには垂直スクロールだけで済む。</li>
+            <li>又は、入れ子にされた各リストを水平スクロールで表示する。入れ子にされた各レベル内のリスト項目の内容を読むには、垂直スクロールだけが必要である。</li>
          </ol>
       </section>
-      <section class="results" id="expected-results"><h3>Expected Results</h3>
+      <section class="results" id="expected-results"><h3>期待される結果</h3>
          <ul>
-            <li>Checks 3 or 4 are true.</li>
+            <li>チェック 3 又は 4 が真である。</li>
          </ul>
       </section>
 
-      <section class="procedure" id="procedure-for-code-snippets"><h3>Procedure for code snippets</h3>
+      <section class="procedure" id="procedure-for-code-snippets"><h3>コードスニペットの事例の手順</h3>
          <ol>
-            <li>Display the web page in a user agent where the page can be zoomed, or the user agent can be resized.</li>
-            <li>Zoom in or resize the browser window so that the viewport is equivallent to 320 CSS pixels wide.</li>
-            <li>The code snippets fits within the width so that it is only necessary so scroll vertically to read each line of code.</li>
-            <li>Or, for code where non-wrapping lines are not essential, the code wraps or a mechanism is provided to allow line wrapping.</li>
-            <li>Or, the indentation and linebreaks are meaningful to the code language. 
+            <li>ウェブページを、ページをズームしたり、ユーザエージェントのサイズを変更したりできるユーザエージェントで表示する。</li>
+            <li>ブラウザのウィンドウをズームイン又はサイズ変更し、ビューポートが 320 CSS ピクセル幅に相当するようにする。</li>
+            <li>コードスニペットは幅に収まるため、各行を読むには垂直スクロールするだけで済む。</li>
+            <li>又は、行の折り返しが必須でないコードでは、コードが折り返されるもしくは、行の折り返しを可能にするメカニズムが提供される。</li>
```

Comment:
> for code where **non-**wrapping lines are not essential
とあるので、訳を調整しました。(二重否定でわかりにくい原文ですが...)

```suggestion
            <li>又は、折り返しのない行が必須でないコードでは、コードが折り返されるもしくは、行の折り返しを可能にするメカニズムが提供される。</li>
```